spec/OpenLineage.yml

```diff
@@ -0,0 +1,212 @@
+openapi: 3.0.2
+info:
+  title: OpenLineage
+  version: 0.0.1
+  description: OpenLineage is an open source **lineage and metadata collection API** for the data ecosystem.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /runs:
+    post:
+      summary: Updates the state of a run
+      description: Updates a run state for a job.
+      operationId: postRunStateUpdate
+      tags:
+        - OpenLineage
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RunStateUpdate'
+      responses:
+        '200':
+          description: OK
+
+components:
+  schemas:
+    RunStateUpdate:
+      type: object
+      properties:
+        id:
+          description: the unique identifier for that run
+          type: string
+          format: uuid
+        transition:
+          description: transitions the run state
+          type: string #TODO: add enum
+          example: 'START|COMPLETE|ABORT|FAIL'
```

Comment:
It may be nice to record multiple state transitions in a single event since the relationship of `runs` -> `run state` is 1-n. However it changes the relationship of an event with the dataset.